src/sections/enterprise-offering/enterprise-bullets/enterprise-bullets.module.scss

```diff
@@ -0,0 +1,13 @@
+.benefits {
+	grid-gap: 40px;
+}
+.icon {
+	margin: 0 auto;
+	height: 50px;
+	width: 50px;
+	margin-bottom: 24px;
+}
+.paragraph {
+  width: 248px;
```

Comment:
the width of the text should be handled by css grid and then you wont need to handle the width or to center the text because the text is aligned to the center

Reply:
But I need to center more the text, without make each box smaller on desktop.
So I removed the width, and just add a padding for screen min-width 920.